### PR TITLE
update: homepage and sponsor details

### PR DESCRIPTION
**Pull request by jemgillam**
### WIP

- Updates to homepage:
  - Grafast and V5 no longer in closed access :clinking_glasses:
  - Button to Grafast website
  - PostGraphile buttons go to postgraphile.org
  - Worker button goes to worker.graphile.org
  - Graphile Build > Graphile Engine
  - New Starter logo (not yet optimized) 

- Updates to sponsor page: 
  - remove references to Patreon
  - Change wording around Grafast - I kept it on this page as the Grafast button on the homepage links to here

### CODE REVIEW

File: src/pages/index.mdx [hunk 1]
```diff
@@ -111,10 +101,10 @@ Migrate.
 
 <div class='df flex-row justify-center' style={{height: '3rem'}}>
   <div class='ph2 df flex-column justify-center'>
-  <a class='button--outline' href='/postgraphile/introduction/'><span class='fas fa-book-open' /> Documentation</a>{ ' '}
+  <a class='button--outline' href='https://postgraphile.org/postgraphile/current/'><span class='fas fa-book-open' /> V4 Documentation</a>{ ' '}
   </div>
   <div class='ph2 df flex-column justify-center'>
-  <a class='button--solid' href='/postgraphile/'><span class='fas fa-arrow-right' /> Learn More</a>{ ' '}
+  <a class='button--solid' href='https:///postgraphile.org'>PostGraphile.org</a>{ ' '}
```
Comment (benjie) [inline src/pages/index.mdx:107]:
```suggestion
  <a class='button--solid' href='https://postgraphile.org'>PostGraphile.org</a>{ ' '}
```

**Comment by jemgillam** [comment]
@benjie please review wording around Grafast on Sponsor page. I wanted to keep something about Grafast there (as the Grafast button links to the sponsor page) , but I don't know if it's too strongly worded. I don't think it quite repeats what is below it, they might be sufficiently different. I'd appreciate your eye before it's all merged.

**Comment by jemgillam** [comment]
I think the new svg needs optimizing, I can't remember how to do that.

**Comment by jemgillam** [comment]
Screenshot 2024-01-18 at 12 46 12

File: src/pages/sponsor.mdx [hunk 2]
```diff
@@ -203,14 +203,15 @@ YOU!**
...
+name={<span>Support Gra<em>fast</em> and PostGraphile V5</span>}
 headline="The next-generation planning and execution engine for GraphQL"
 >
 
-Gra<em>fast</em> and PostGraphile Version 5 are available as a preview to
-Graphile sponsors; sign up as a sponsor to get access to the binaries and
-documentation (sponsor at the Featured Tier for access to the repository, issues
-and support!)
+Gra<em>fast</em> and PostGraphile Version 5 are now in beta; 
+this timely release was in large part due to the support given through sponsorship.
```
Comment (benjie) [inline src/pages/sponsor.mdx:211]:
```suggestion
Gra<em>fast</em> and PostGraphile Version 5 are now in open beta thanks
in large part to the support given through sponsorship.
```

File: src/pages/sponsor.mdx [hunk 3]
```diff
@@ -203,14 +203,15 @@ YOU!**
...
 
-Gra<em>fast</em> and PostGraphile Version 5 are available as a preview to
-Graphile sponsors; sign up as a sponsor to get access to the binaries and
-documentation (sponsor at the Featured Tier for access to the repository, issues
-and support!)
+Gra<em>fast</em> and PostGraphile Version 5 are now in beta; 
+this timely release was in large part due to the support given through sponsorship. 
+
+Our sponsors enable us to continue to concentrate on our projects with the freedom to work on 
+our software in the most effective way &mdash; without diverting our resources to other endeavors.
```
Comment (benjie) [inline src/pages/sponsor.mdx:214]:
```suggestion
Sponsorship is the most effective way to support our open source projects &mdash; it enables
us to invest time directly into their maintenance and advancement without adding additional
time burdens.
```

**Comment by jemgillam** [comment]
I've committed your suggestions. I think in context of the whole page it still reads well - the bit about Grafast & Version 5 is highlighting one of the recent uses of sponsorship money.

**Review by benjie — APPROVED** [review]

**Pull request merged**